joomla/joomla-cms
# [CODEMIRROR] Two new font options, highlight selection matches.

#### Summary of Changes

CodeMirror updated to 5.14.2 and additional addons bundled.
Plugin has a new option to highlight the selected word.
Two new font options: Roboto Mono and Rubik Mono One - one of them is pretty good, the other is awful. You decide!
#### Testing Instructions

Go to the codemirror plugin options and switch on `Highlight Selection Matches`. Then, edit some content with codemirror. Instances of the word your cursor is on should be highlighted throughout the document.
